alert-policies/lampy/TransactionErrors.yml

```diff
@@ -0,0 +1,36 @@
+name: Transaction Errors
+
+description: |+
+  This alert is triggered when the transactions fail more than 10% of the time in 5 minutes.
```

Comment:
can be changed to: `This alert is triggered when the transactions failure rate exceeds 10% for 5 minutes.`